### PR TITLE
DACT-928: Update director check your answers screen

### PR DESCRIPTION
**Pull request by anona-w**
https://companieshouse.atlassian.net/browse/DACT-928 - CH01 Check your answers screen

### CODE REVIEW

File: src/controllers/update/update.director.check.answers.controller.ts [hunk 1]
```diff
@@ -38,6 +45,27 @@ export const post = async (req: Request, res: Response, next: NextFunction) => {
...
+
+    // Check if ETAG matches
+    if (currentOfficerFiling.referenceEtag !== companyAppointment.etag) {
+      return res.redirect(
+        urlUtils.setQueryParam(urlUtils.getUrlToPath(BASIC_STOP_PAGE_PATH, req), URL_QUERY_PARAM.PARAM_STOP_TYPE, STOP_TYPE.ETAG));
+    }
+
+    // Run full api validation
+    const validationStatus = await getValidationStatus(session, transactionId, submissionId);
+    console.info(`ANONA Validation status: ${JSON.stringify(validationStatus)}`);
```
Comment (mdesaikns) [inline src/controllers/update/update.director.check.answers.controller.ts:65]: leaving breadcrumbs of your name here I see 🥇

File: views/update/update-director-check-answers.html [hunk 2]
```diff
@@ -18,6 +18,9 @@
       {% include "includes/list/errors.html" %}
       
       <form action="" method="post">
+        <span class="govuk-caption-l gem-c-title__context">
+          {{name}}
```
Comment (mdesaikns) [inline views/update/update-director-check-answers.html:22]: is this leftover from testing?

Comment (anona-w) [reply views/update/update-director-check-answers.html:22]: No the prototype had the directors name at the top of the page so this was to add that - reusing the existing variable "name" which did the formatting etc

**Review by mdesaikns — APPROVED** [review]

**Review by clewis1 — APPROVED** [review]
Looks good.

**Review by clewis1 — APPROVED** [review]

**Pull request merged**